src/logic.js

```diff
@@ -1,33 +1,34 @@
-const { Rune } = window;
+const { Rune } = window
 
 Rune.initLogic({
   minPlayers: 4,
   maxPlayers: 4,
-  setup: () => {
+  setup: (allPlayerIds) => {
+    const scores = {}
+    for (let playerId of allPlayerIds) {
+      scores[playerId] = 0
+    }
```

Comment:
This is the main change, ignore the other changes where my linter removed all the semicolons lol.